bin/install-dev-osx.sh

```diff
@@ -86,5 +86,5 @@ fi
 
 # Install testing dependencies
 echo "Installing testing dependencies"
-pip install -r "test/integration_tests/requirements-integration-test.txt"
+pip install --no-cache-dir -r "test/integration_tests/requirements-integration-test.txt"
```

Comment:
It's actually just to align with another `pip` command above, to ensure consistent behavior.